kbs/src/api/src/http/resource.rs

```diff
@@ -243,3 +230,32 @@ pub(crate) fn jwe(tee_pub_key: TeePubKey, payload_data: Vec<u8>) -> Result<Respo
         tag: "".to_string(),
     })
 }
+
+pub(crate) fn get_pkey(claims: Value) -> Result<TeePubKey> {
+    let pkey_value = claims
+        .get("customized_claims")
```

Comment:
Can we define/reuse a type `CustomizedClaim` instead of manually performing the parsing by stepping through properties?